ios/FluentUI/Other Cells/BooleanCell.swift

```diff
@@ -40,9 +48,11 @@ open class BooleanCell: TableViewCell {
     ///   - title: Text that appears as a single line
     ///   - customView: An optional custom view that appears on the leading edge, adjacent to the start of the `title` text
     ///   - isOn: A boolean value describing the `isOn` state of the accessory view
-    @objc open func setup(title: String, customView: UIView? = nil, isOn: Bool = false) {
+    ///   - isSwitchEnabled: A boolean value describing the `isEnabled` state of the accessory view
+    @objc open func setup(title: String, customView: UIView? = nil, isOn: Bool = false, isSwitchEnabled: Bool = true) {
```

Comment:
can we create a convenience method that doesn't take in this new optional variable? the reason I ask this, is that the existing objc caller(clients) need to update the new param otherwise when they upgrade the library version it will have a compiler error.